databus-client2/src/main/java/com/bazaarvoice/emodb/databus/client2/client/DatabusClient.java

```diff
@@ -0,0 +1,553 @@
+package com.bazaarvoice.emodb.databus.client2.client;
+
+
+import com.bazaarvoice.emodb.auth.apikey.ApiKeyRequest;
```

Comment:
nitpick, sort imports - https://blog.jetbrains.com/webstorm/2018/05/optimize-imports-in-webstorm/